src/main/java/org/example/DoWhileLoopMatrixGenerator.java

```diff
@@ -0,0 +1,39 @@
+package org.example;
+
+
+import static java.lang.Math.abs;
+
+public class DoWhileLoopMatrixGenerator {
+    public static void generateAndPrintMatrixWithDoWhileLoop(int n) {
```

Comment:
according to Java style there should be an empty line after the class declaration